src/toil/common.py

```diff
@@ -490,15 +490,21 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         """
-        Clean up after a workflow invocaton. Depending on the configuration, delete the job store.
+        Clean up after a workflow invocation. Depending on the configuration, delete the job store.
         """
+        try:
+            if (exc_type is not None and self.config.clean == "onError" or
+                            exc_type is None and self.config.clean == "onSuccess" or
+                        self.config.clean == "always"):
+                logger.info("Attempting to delete the job store")
+                self._jobStore.deleteJobStore()
+                logger.info("Successfully deleted the job store")
+        except Exception as e:
+            if exc_type is None:
+                raise
+            else:
+                logger.exception(e)
```

Comment:
Please read the documentation on this method. You are using it wrong. The log message should give some context.